src/main/java/mcp/mobius/betterbarrels/common/blocks/TileEntityBarrel.java

```diff
@@ -669,9 +669,10 @@ public ItemStack getStackInSlot(int islot) {
 	public ItemStack decrStackSize(int islot, int quantity) {
 		TileEntity ent = this.worldObj.getTileEntity(this.xCoord, this.yCoord - 1, this.zCoord);
 		ItemStack stack;
-		if (ent instanceof TileEntityHopper) {
+        //TODO Kotl Remove Code fix dupe
+        /*if (ent instanceof TileEntityHopper) {
```

Comment:
Why that code was there in the first place? Isn't it there to fix the same issue that storage drawers had?

Reply:
I don't know why this code is there, but the fact is that with it there is a dupe without it.

Reply:
removing this code did not cause any negative results